### DISCUSSION TITLE
How to run the script in relative directory?

### DISCUSSION THREAD

**Discussion by vvizden** [pnpm/pnpm#4062] [Q&A] [answered]
Hi, when I try `pnpm run dev --filter ../packageRootDir`, I get the message `No projects matched the filters`. How can i run the script in the relative directory?

**Comment by Jack-Works** [comment, marked as answer]
Maybe `pnpm -C path run ...`

**Reply by vvizden** [reply]
Thanks, Maybe I need to read the pnpm docs carefully!
